backend/lambdas/generators/json_report/tests/test_generate_report.py

```diff
@@ -410,6 +432,28 @@ def test_get_inventory_report_for_technology_discovery(self):
         inventory = get_inventory(mock_scan)
         self.assertEqual(expected_inventory, inventory)
 
+    def test_get_configuration_report_for_repo_health(self):
+        expected_configuration = PLUGIN_RESULTS(
+            {
+                "repo_health": [
+                    {
+                        "type": "branch_commit_signing",
+                        "name": "Branch - Require Commit Signing",
+                        "description": "Branch protection rule is enabled to enforce code signing",
+                        "pass": True,
+                    }
+                ]
+            },
+            PluginErrors(),
+            True,
+            {"repo_health": 1},
+        )
+
+        mock_scan = unittest.mock.MagicMock(side_effect=Scan())
+        mock_scan.pluginresult_set.filter.return_value = [TEST_REPO_HEALTH]
+        configuration = get_configuration(mock_scan)
+        self.assertEqual(expected_configuration, configuration)
```

Comment:
should there also be a test that checks a failing config result and that it sets scan `success` = `false`

Reply:
I've added the test here: backend/engine/tests/test_plugin_repo_health.py